server.js

```diff
@@ -26,11 +27,13 @@ async function createPayment(req, res) {
   if (!validatePaymentPayload(payload)) {
     throw createError(400, 'Bad Request');
   }
+
+  const idempotencyKey = payload.idempotencyKey || crypto.randomUUID();
```

Comment:
`payload.idempotencyKey` is a holdover from previous iterations and can be removed from this file and `schema.js`

Reply:
Thanks for the callout - went ahead and removed it. I also added a comment linking to Square's documentation about idempotency